client/my-sites/hosting/site-backup-card/index.js

```diff
@@ -36,7 +33,7 @@ const SiteBackupCard = ( {
 			requestBackups( siteId );
 			setIsLoading( true );
 		}
-	}, [ disabled, isLoading, lastGoodBackup, siteId ] );
+	}, [ disabled, hasRetrievedLastBackup, isLoading, lastGoodBackup, requestBackups, siteId ] );
```

Comment:
Given that `requestBackups` is a function that doesn't change, I don't think we need to pass it here.

Reply:
I believe it does change in some odd circumstances (@jsnajdr would know more) but in general you're right, we don't need to. It's harmless to add it, though, and it does fix the lint error that its absence was causing.